integration-tests/opentelemetry-quartz/src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+quarkus.application.name=opentelemetry-quartz-instrumentation-it
+quarkus.application.version=999-SNAPSHOT
+
+disabled=disabled
```

Comment:
What are these properties for?

Reply:
I borrowed these top properties from other OTel integration tests, but it should not affect the test's outcome.